bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TopBNV/BNV_run_card.dat

```diff
@@ -0,0 +1,266 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************
+# Running parameters
+#*******************
+#
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+tag_1	=  run_tag  ! name of the run
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+True	=  gridpack   !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+10000 =  nevents  ! Number of unweighted events requested
```

Comment:
hi not an error but 10000 events is meaningless for a pilot run, might want to reduce it to 2K

Reply:
@rgoldouz Just to comment here, I haven't changed this and merged already since it seems you were in a hurry. But it would help at least a little bit if you just reduce the nevents to some smaller number like 2K when you make the gridpacks